client/secret-operations.go

```diff
@@ -52,3 +52,27 @@ func (client Client) GetPlainTextSecretsViaServiceToken(secretFolderPath string,
 
 	return plainTextSecrets, &serviceTokenDetails, nil
 }
+
+func (client Client) GetRawSecretsViaMachineIdentity(secretFolderPath string, envSlug string, workspaceId string) ([]RawV3Secret, error) {
```

Comment:
Nit: let's keep the name more generic since raw ends points aren't only bound to machine identity. I think GetRawSecrets would work